src/main/java/com/training/springpractice/models/CatalogBody.java

```diff
@@ -0,0 +1,58 @@
+package com.training.springpractice.models;
```

Comment:
Is this CatalogBody an Entity?
Because I see an @Id and some other Entity annotations here but I don't see @Entity annotation, please review how this class should work.